client/src/ui/Login.jsx

```diff
@@ -28,32 +29,33 @@ export default class Login extends Component {
 
     this.defaultState = this.state;
   }
-  
+
   componentWillMount() {
     // The following is used to show admin panel if a user's token is found to be an admin
-    if(Session.get("token") != ""){
-      Meteor.call('tokenIsAdmin', Session.get("token"), (error, result) => {
-        if(result){
+    if (Session.get("token") != "") {
+      Meteor.call("tokenIsAdmin", Session.get("token"), (error, result) => {
+        if (result) {
           Session.set("adminlogin", true);
-          this.setState({executeLogin: false}); //This isn't necessary as it should alrady be allFalse
-                                                // but it is used to refresh the render()
-        }
-        else{
-          this.setState({executeLogin: true});
+          this.setState({ executeLogin: false }); //This isn't necessary as it should alrady be allFalse
+          // but it is used to refresh the render()
+        } else {
+          this.setState({ executeLogin: true });
         }
       });
-    }
-    else{
-      this.setState({executeLogin: true});
+    } else {
+      Session.set("adminlogin", true); // OVERRIDEN
+      this.setState({ executeLogin: false });
+  
+      console.log("LOGIN WAS OVERRIDDEN ON THIS BRANCH.");
```

Comment:
is this fixed?

Reply:
I was testing locally over HTTP, which never works correctly with Google Auth. It should work fine last time I checked, just have to setup an HTTPS proxy and set myself as admin in the DB.